src/library/browse/browsefeature.cpp

```diff
@@ -288,8 +288,8 @@ void BrowseFeature::onRightClickChild(const QPoint& globalPos, QModelIndex index
 namespace {
 // Get the list of devices (under "Removable Devices" section).
 QList<TreeItem*> getRemovableDevices(LibraryFeature* pFeature) {
-#if defined(__WINDOWS__)
     QList<TreeItem*> ret;
+#if defined(__WINDOWS__)
```

Comment:
what is this change?  Is it just your master being out of date?

Reply:
The placement and usage of the recently introduced #ifdefs in browsefeature.cpp was inadequate and caused warnings for macOS that doesn't support removable drives like Windows and Linux.

Reply:
got it, thanks!  ok to merge